src/components/Modeler.vue

```diff
@@ -514,9 +514,11 @@ export default {
       store.commit('highlightNode', this.processNode);
     },
     removeUnsupportedElementAttributes(definition) {
-      const unsupportedElements = ['documentation', 'extensionElements'];
       unsupportedElements.filter(name => definition.get(name))
         .forEach(name => definition.set(name, null));
+
+      definition.$descriptor.properties = definition.$descriptor.properties.filter(
+        p => unsupportedElements.indexOf(p.name) === -1);
```

Comment:
I think this could be a little clearer as follows:
```suggestion
        p => !unsupportedElements.includes(p.name));
```